.yamato/scripts/build_osx.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+set -e
+
+# Usage: build_osx.cmd <Architecture> <Configuration>
+#
+#   Architecture is one of arm64 or x64 - x64 is the default if not specified
+#   Configuration is one of Debug or Release - Release is the default is not specified
+#
+# To specify Configuration, Architecture must be specified as well.
+
+if [ "$1" = "" ]; then
+    architecture=x64
```

Comment:
Bonus points if this could check the system architecture and build that instead of defaulting to x64

Reply:
It looks like `uname -p` might work for this. What does that command return on your M1 machine?

Reply:
I've done this at c2797aaa2b2e28b26ccfe8eb6acb2b6959d0761c.